special-pages/pages/new-tab/app/entry-points/omnibar.js

```diff
@@ -1,11 +1,14 @@
 import { h } from 'preact';
 import { Centered } from '../components/Layout.js';
 import { OmnibarCustomized } from '../omnibar/components/OmnibarCustomized.js';
+import { PersistentTermsProvider } from '../omnibar/components/PersistentTermsProvider.js';
 
 export function factory() {
     return (
         <Centered data-entry-point="omnibar">
-            <OmnibarCustomized />
+            <PersistentTermsProvider>
```

Comment:
This state needs to live for the lifespan of the page, not the widget, so we go 1 level higher